aws/sagemaker_studio/README.md

```diff
@@ -0,0 +1,45 @@
+# Instructions for Running RAPIDS + SageMaker Studio + HPO
+
+0. Upload train/test data to S3 
+
+   - We offer a dataset for the HPO demo in a public bucket hosted in either the `us-east-1` or `us-west-2` regions:
+   > https://s3.console.aws.amazon.com/s3/buckets/sagemaker-rapids-hpo-us-east-1/   
+   > https://s3.console.aws.amazon.com/s3/buckets/sagemaker-rapids-hpo-us-west-2/
+
+1. Publish a container to the Amazon Elastic Container Registry (ECR)
+
+   - Because SageMaker Studio runs from a Docker container, you cannot create and upload a Docker container from a SageMaker Studio instance. However, in order to use a custom container with an Estimator, you must first publish it to the ECR. In order to work around this, we recommend running the studio_hpo_container_setup.ipynb notebook within a notebook instance. To create a notebook instance: 
```

Comment:
Just looking through this again how about this CLI for building images in SageMaker Studio? https://aws.amazon.com/blogs/machine-learning/using-the-amazon-sagemaker-studio-image-build-cli-to-build-container-images-from-your-studio-notebooks/

Reply:
I still need to get a few more permissions to execute sm-build but will keep you posted